### PR TITLE
Add yarn in stock

### PR DESCRIPTION
**Pull request by MichanNan**
add yarn in stock, image upload also works 😄

### CODE REVIEW

File: components/YarnStockForm/index.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          radius="1rem"
+          name="skein"
+          backgroundColor="#f5f5f5"
+        />
+
+        <ButtonContainer>
+          <StyledButton
+            width="5rem"
+            height="2rem"
+            type="cancel"
```
Comment (rueveydaakuel) [inline components/YarnStockForm/index.js]: I am not sure if `type="cancel"` is supported by react?

**Review by rueveydaakuel — COMMENTED** [review]
Hi Nan, it looks totally fine to me; must add that I really like your Project 🥳
I just left a little comment because I wasn't sure about it.

File: components/YarnStockForm/index.js (hunk 1 above)
Comment (MichanNan) [reply components/YarnStockForm/index.js]: thanks Rüveyda! I also found it is unnecessary to have type for the button here.

File: components/Stock/YarnStockForm/index.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+          radius="1rem"
+          name="skein"
+          backgroundColor="#f5f5f5"
+        />
+
+        <ButtonContainer>
+          <StyledButton
+            width="8rem"
+            height="3rem"
+            type="cancel"
```
Comment (ryliecc) [inline components/Stock/YarnStockForm/index.js:99]: you can use `type="button"` here. cancel is not a button type. See [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attributes) for reference.

**Review by ryliecc — CHANGES_REQUESTED** [review]
Other than the button type the code looks good to me. Well done! 🤗

File: components/YarnStockForm/index.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ data }),
+    });
+
+    router.push("/");
+    mutate();
+  }
```
Comment (matschi3) [inline components/YarnStockForm/index.js]: not sure if it is sth to mention, but maybe switch `router.push("/")` and `mutate()` to have the reRender happen before the user gets back to `"/"`.
But possible, that my thinking is wrong

File: components/Icon/BackIcon.js [hunk 4]
```diff
@@ -12,4 +12,5 @@ export default function BackIcon({ handleGoBack }) {
 const BackItem = styled.div`
   position: absolute;
   left: 2.5rem;
+  z-index: 99;
 `;
```
Comment (matschi3) [inline components/Icon/BackIcon.js:16]: i assume you know, its best practice to avoid using z-index ✌🏼

**Comment by matschi3** [comment]
Pretty clean. LGTM 🚀

File: components/YarnStockForm/index.js (hunk 3 above)
Comment (MichanNan) [reply components/YarnStockForm/index.js]: Hi i am also not sure about that, but it makes sense that first mutate then push to other page. i will change it by next PR thanks!

**Pull request merged**